src/main/resources/init_db.sql

```diff
@@ -12,3 +12,41 @@ ALTER TABLE `taxi_service`.`manufacturers`
 
 ALTER TABLE `taxi_service`.`manufacturers`
     RENAME COLUMN `delete` TO `deleted`;
```

Comment:
let's put into this file the `CREATE TABLE` statements that are correct from the start

Reply:
done